docs/Files/UI/Solution-Manager/Library/EscalationManager/Escalation-Manager.md

```diff
@@ -0,0 +1,17 @@
+---
+lang: en-us
+title: Managing Escalation Manager
+viewport: width=device-width, initial-scale=1.0
+---
+
+# Overview
+
+Escalation Manger has options related to escalations.
```

Comment:
escalations should be capitalize - we typically capitalize when referring to OpCon objects.

Reply:
Fixed